build.sc

```diff
@@ -49,6 +51,10 @@ object core extends Module {
   object jvm extends Cross[CoreJvm](ScalaVersions.all: _*)
   object js  extends Cross[CoreJs](ScalaVersions.all: _*)
 }
+object `sbt-maven-repository` extends Module {
```

Comment:
Added new module `coursier-sbt-maven-repository` containing `SbtMavenRepository`, `SbtPom` and `SbtPomParser`

`coursier-cli` depends on this new artifact.